sycl/doc/extensions/experimental/sycl_ext_intel_data_flow_pipes_properties.asciidoc

```diff
@@ -144,7 +145,7 @@ struct first_symbol_in_high_order_bits_key {
 
 enum class protocol_name : /* unspecified */ {
   avalon_streaming = 0,
-  avalon_mm = 1,
+  implement_in_csr = 1,
```

Comment:
Given that this is already implemented with the old name and that we have existing tests and customers using it I'd rather we minimize churn and only change it once rather than twice.  And at that time, let's change both the spec and the headers together.

Reply:
Updated the name back to `avalon_mm` for the time being.